src/dmd/mars.d

```diff
@@ -486,6 +486,11 @@ private int tryMain(size_t argc, const(char)** argv)
         message("binary    %s", global.params.argv0);
         message("version   %s", global._version);
         message("config    %s", global.inifilename ? global.inifilename : "(none)");
+        OutBuffer buf;
```

Comment:
I recommend adding a header comment like "Print DFLAGS environment variable" for this block of code.